src/Illuminate/Console/Command.php

```diff
@@ -321,36 +332,71 @@ public function table(array $headers, $rows, $style = 'default')
     }
 
     /**
+     * Check if verbosity level is in bounds.
+     *
+     * @param  string|int  $level
+     * @return bool
+     */
+    public function inVerbosity($level)
+    {
+        if (! is_int($level) && isset($this->verbosity[$level])) {
+            $level = $this->verbosity[$level];
+        } else {
+            $level = OutputInterface::VERBOSITY_NORMAL;
```

Comment:
Wrong condition. `OutputInterface::VERBOSITY_VERY_VERBOSE` will be overwritten.
````php
if (isset($this->verbosity[$level])) {
    $level = $this->verbosity[$level];
} elseif (! is_int($level)) {
    $level = OutputInterface::VERBOSITY_NORMAL;
}
````